packages/odyssey-react-mui/src/labs/SideNav/SideNavHeader.tsx

```diff
@@ -93,7 +95,7 @@ const SideNavHeader = ({
       </SideNavLogoContainer>
 
       <SideNavHeadingContainer odysseyDesignTokens={odysseyDesignTokens}>
-        <Heading6 component="h2">{isLoading ? <Skeleton /> : appName}</Heading6>
+        <Heading5 component="h2">{isLoading ? <Skeleton /> : appName}</Heading5>
```

Comment:
I'm so curious: why Heading5 w/ component h2? I know this predates your change.

Reply:
`Heading5` here represents the visual style. While `component="h2"` correlates to the underlying semantic HTML tag. So, an `h2` that has the same style as our `h5` component

Reply:
That being said, these likely can be just `p` tags

Reply:
But, this is the app name so, maybe we want it as a heading tag?

Reply:
Our scenario is sort of like "Example 2" here: https://www.w3.org/WAI/tutorials/page-structure/headings/
So, `h2` probably makes sense